test/task.h

```diff
@@ -184,6 +184,7 @@ enum test_status {
 #  define inline __inline
 # endif
 
+# if defined(_MSC_VER) && _MSC_VER < 1900
```

Comment:
Oh darn. Then maybe a version check. Because at some point they are going to retire the non-standard stuff and then again someone will have to deal with it. :)